src/index.html

```diff
@@ -11,8 +11,44 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
```

Comment:
You have linked a SCSS file in your HTML. SCSS is a preprocessor language that needs to be compiled into CSS for browsers to understand. You should link the compiled CSS file instead.